vm/src/stdlib/os.rs

```diff
@@ -1455,6 +1455,76 @@ mod _os {
         }
     }
 
+    #[pyattr]
+    #[pyclass(module = "os", name = "times_result")]
+    #[derive(Debug, PyStructSequence)]
+    struct TimesResult {
+        pub user: f64,
+        pub system: f64,
+        pub children_user: f64,
+        pub children_system: f64,
+        pub elapsed: f64,
+    }
+
+    #[pyimpl(with(PyStructSequence))]
+    impl TimesResult {}
+
+    #[pyfunction]
+    fn times(vm: &VirtualMachine) -> PyResult {
```

Comment:
This doesn't really matter, but note that you could have this just return `PyResult<TimesResult>` - anything that implements `IntoPyObject` can be returned from a function and it will get automatically converted to a PyObjectRef.